### PR TITLE
Updates README files

### PR DESCRIPTION
**Pull request by lucianomartin**
The list of guidelines can be found in https://xmosjira.atlassian.net/wiki/spaces/PR/pages/2197749954/Read+Me+File+Guidelines

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -29,47 +29,10 @@ Features
...
-    - locals: interface i2c_slave_callback_if i;
-    - fn: i2c_slave(i, p_scl, p_sda, 0);
-    - pins: 2
-    - ports: 2 (1-bit)
-
 Software version and dependencies
 .................................
 
-.. libdeps::
+For a list of direct dependencies, look for DEPENDENT_MODULES in lib_xxx/module_build_info.
```
Comment (mbanth) [inline README.rst:35]: Replace` lib_xxx` with `lib_i2c`

**Review by mbanth — CHANGES_REQUESTED** [review]
One small change.

File: README.rst (hunk 1 above)
Comment (lucianomartin) [reply README.rst]: Fixed. Thanks for spotting it.

**Review by mbanth — APPROVED** [review]
Looks good.

**Comment by lucianomartin** [comment]
This PR has been replaced by https://github.com/xmos/lib_i2c/pull/65

**Pull request closed without merging**